http4s-blaze-client/src/main/scala/com/avast/server/toolkit/http4s/Http4sBlazeClient.scala

```diff
@@ -0,0 +1,52 @@
+package com.avast.server.toolkit.http4s
+
+import cats.effect.{ConcurrentEffect, Resource, Sync}
+import cats.implicits._
+import cats.{Applicative, Traverse}
+import com.avast.server.toolkit.ssl.{SslContextConfig, SslContextModule}
+import javax.net.ssl.SSLContext
+import org.http4s.client.Client
+import org.http4s.client.blaze.BlazeClientBuilder
+
+import scala.concurrent.ExecutionContext
+import scala.language.higherKinds
+
+object Http4sBlazeClient {
+
+  /** Makes [[org.http4s.client.Client]] (Blaze) initialized with the given config. */
+  def make[F[_]: ConcurrentEffect](config: Http4sBlazeClientConfig, executionContext: ExecutionContext): Resource[F, Client[F]] = {
+    makeClient(config, executionContext)
+  }
+
+  private def makeClient[F[_]: ConcurrentEffect: Applicative](config: Http4sBlazeClientConfig,
+                                                              executionContext: ExecutionContext): Resource[F, Client[F]] = {
+    for {
+      maybeSslContext <- Resource.liftF(sslContext(config.sslContext))
```

Comment:
maybe the `maybeSslContext` shouldn't be created here, but come as the parameter to the `make` function

Reply:
It is an option but prefer it this way - it is part of the entire server configuration.

Reply:
I don't object having a method which creates `SSLContext` behind the scene.
but at the same time, I think we should have a `make...` method which takes a custom `SSLContext` instance

Reply:
Sure, that makes sense but let's add it once we need it. I don't want to prolong this PR anymore.